plugins/org.jkiss.dbeaver.data.office/OSGI-INF/l10n/bundle_zh.properties

```diff
@@ -17,6 +17,8 @@ dataTransfer.processor.xlsx.property.splitSqlText.name = \u5207\u5206 SQL \u6587
 dataTransfer.processor.xlsx.property.splitSqlText.description = \u7528 CR \u6309\u884C\u5207\u5206\u5BFC\u51FA\u7684 SQL
 dataTransfer.processor.xlsx.property.headerFont.name = \u6807\u9898\u884C\u5B57\u4F53
 dataTransfer.processor.xlsx.property.headerFont.description = \u9996\u884C\u5B57\u4F53\u5C5E\u6027
+dataTransfer.processor.xlsx.property.headerFontName.name=\u000d\u000a\u6807\u9898\u5b57\u4f53\u540d\u79f0
+dataTransfer.processor.xlsx.property.headerFontName.description=\u7b2c\u4e00\u884c\u5b57\u4f53\u540d\u79f0
```

Comment:
Until you're sure that you know that language, such translations are useless and tend to contain grammatical errors.